apps/condo/domains/user/schema/AuthenticateUserThroughExternalIdentityService.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Generated by `createservice user.AuthenticateUserThroughExternalIdentityService`
+ */
+const { isNil } = require('lodash')
+
+const { GQLCustomSchema, getSchemaCtx } = require('@open-condo/keystone/schema')
+const access = require('@condo/domains/user/access/AuthenticateUserThroughExternalIdentityService')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+
+
+const { UserExternalIdentity, User } = require('@condo/domains/user/utils/serverSchema')
+const { getIdentityIntegration } = require('@condo/domains/user/integration/identity')
+const { IDENTITY_INTEGRATION_DOES_NOT_SUPPORT_LOGIN_FORM, USER_NOT_FOUND } = require('@condo/domains/user/constants/errors')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const errors = {
+    IDENTITY_INTEGRATION_DOES_NOT_SUPPORT_LOGIN_FORM: {
+        mutation: 'authenticateUserThroughExternalIdentity',
+        variable: ['data', 'identityType'],
+        code: BAD_USER_INPUT,
+        type: IDENTITY_INTEGRATION_DOES_NOT_SUPPORT_LOGIN_FORM,
+        message: 'Identity integration does not support login form',
+    },
+    USER_NOT_FOUND: {
+        mutation: 'authenticateUserThroughExternalIdentity',
+        code: BAD_USER_INPUT,
+        type: USER_NOT_FOUND,
+        message: 'Unable to find user by provided external identity. Try to register',
+        variable: ['data', 'tokenSet'],
+        messageForUser: 'api.user.authenticateUserThroughExternalIdentity.USER_NOT_FOUND',
+    },
+}
+
+const AuthenticateUserThroughExternalIdentityService = new GQLCustomSchema('AuthenticateUserThroughExternalIdentityService', {
+    types: [
+        {
+            access: true,
+            type: 'input AuthenticateUserThroughExternalIdentityInput { identityType: IdentityType!, tokenSet: JSON! }',
```

Comment:
why JSON?

Reply:
tokenSet is highly depends by IdentityIntegration system. That means we don't know the structure of this parameter, since it comes from external system